enhancements/network/egressip_for_ovn_kubernetes.md

```diff
@@ -0,0 +1,134 @@
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+EgressIP is currently a feature for openshift-sdn and is planned to be implemented for OVN-Kubernetes and subsequently integrated in the 4.6 release of OpenShift. This feature implementation for OVN-Kubernetes requires feature parity with openshift-sdn, but also aims for a feature improvement which is detailed in this document. 
+
+### Background
+
+EgressIP, as it's currently implemented for openshift-sdn, allows traffic exiting a cluster namespace to have a fixed IP address. This IP is attached to the primary network interface on one of the cluster nodes. The assignment of this IP for openshift-sdn is a two step process. The cluster admin first needs to modify the `hostsubnet` object of the nodes he/she would like to act as "egress nodes", they do so by specifying `EgressIPs` or `EgressCIDRs` for that `hostsubnet`. Secondly they need to modify the `netnamespace` object with the field `EgressIPs`, which finally tells openshift-sdn that those `EgressIPs` for that namespace need to be assigned between the nodes the cluster admin modified in step one. The IP address can be manually assigned or left up to the network plugin to automatically assign. For automatic assignment to be done, the user needs to set `EgressCIDRs` on the `hostsubnet` object, if `EgressIPs` are instead set: the assignment is left completely to the user's own management. Egress IP is managed in a highly available manner if automatic assignment is chosen, meaning: if the node currently hosting that IP falls off the cluster, the network plugin re-assigns the egress IP to another cluster node as a fallback mechanism.
+
+This enhancement proposal focuses on simplifying this process as well as adding a `podSelector` to the `EgressIP` object, allowing a user to specify a subset of pods within the namespace which should have an egress IP defined. 
+
+If we resume what openshift-sdn concretely achieves by its two-step process, we have the following: 
+
+1.1. It allows the cluster admin to have a direct say on what nodes act as egress nodes.
+1.2. The final assignment of egress IPs is still a manual process, and even though the re-distribution of them, warrented by the HA mechanism, is automatic (in case of a node failure), the newly distributed egress IP will remain the same. 
+
+All in all, the feature is quite complex as it introduces a lot of notions which obfuscate these two final functionalities/goals.  
+
+## Motivation
+
+The motivation behind this requirement comes from partially from product management (assigning a `podSelector`  and essentially allowing cluster admins the freedom to control cluster egress traffic on a lower granular level), and partially from us in the networking team (simplifying the usability for cluster admins and our maintenance of the code base which implements it).   
```

Comment:
(Again, the _primary_ motivation here is "making ovn-kubernetes have feature parity with openshift-sdn". This is all secondary points on top of that.)

OK, I guess you're writing this from the perspective of what's being added on top of the work you already did that hasn't merged yet, but you should be writing it to describe the entire feature.

Reply:
Will re-phrase as such